src/db/seeds/Stalls.csv

```diff
@@ -1,4 +1,4 @@
-﻿name,description,rating,contactNo,unitNo,hawkerCentreId
+﻿name,description,rating,contactNo,unitNo,openingHours,hawkerCentreId
 Selera Rasa Nasi Lemak,"Serving traditional Nasi Lemak since 1998, when The Undertaker threw Mankind off Hell In A Cell, and plummeted 16 ft through an announcer's table.",4,97654321,#01-02,1

```

Comment:
Perhaps there should be some sample data for opening hours?

Reply:
ok I'll update